torch/optim/lr_scheduler.py

```diff
@@ -1447,7 +1450,10 @@ def __init__(
         base_lrs = self._format_param("base_lr", optimizer, base_lr)
         if last_epoch == -1:
             for lr, group in zip(base_lrs, optimizer.param_groups):
-                group["lr"] = lr
+                if isinstance(group["lr"], Tensor):
+                    group["lr"].fill_(lr)
```

Comment:
are we guaranteed here that lr will never be a Tensor?

Reply:
I think we are, but I don't think there's harm in extracting the value out, let me do that.